CRM/Utils/Hook.php

```diff
@@ -309,6 +309,10 @@ public function runHooks(
    * @param $moduleList
    */
   public function requireCiviModules(&$moduleList) {
+    foreach ($GLOBALS['CIVICRM_👻_MODULES'] ?? [] as $prefix) {
```

Comment:
really? a ghost?

Reply:
Iit makes it less likely that downstream will use it in unintended ways... because the name is elusive...

Reply:
Or do you think this variable might come back to haunt us?

Reply:
> It makes it less likely that downstream will use it in unintended ways... because the name is elusive...

or perhaps the name is 'ghostly'